content/sensu-go/5.20/reference/tokens.md

```diff
@@ -27,15 +28,30 @@ The agent replaces any tokens with matching attributes from the entity definitio
 Invalid templates or unmatched tokens return an error, which is logged and sent to the Sensu backend message transport.
 Checks with token-matching errors are not executed.
 
-Token substitution is supported for [check definition][7] `command` attributes and [hook][8] `command` attributes.
+Token substitution is supported for [check][7], [hook][8], [handler][10], and [mutator][11] `command` attributes.
 Only [entity attributes][4] are available for substitution.
+Token substitution is not available for event filters because filters already have access to the entity.
+
 Available attributes will always have [string values][9], such as labels and annotations.
 
 ## Manage entity labels
 
 You can use token substitution with any defined [entity attributes][4], including custom labels.
 See the [entity reference][6] for information about managing entity labels for proxy entities and agent entities.
 
+## Manage asset changes
+
+Use token substitution to retrieve changes to assets without manually updating your check, hook, handler, and mutator commands.
+
+For example, if an asset changes, the asset's specific path on disk will change.
```

Comment:
Same as ^. Not sure if we should document the asset local path in a subsequent PR once we have the actual implementation.

Reply:
yes, I think you are right if that will not be part of 5.20.